pkg/util/ip/addr.go

```diff
@@ -43,3 +44,31 @@ func lastIP(subnet net.IPNet) net.IP {
 	}
 	return end
 }
+
+func IPsToStrings(ips []configv1.IP) []string {
+	res := []string{}
```

Comment:
don't we know the size of `res` from the size of `ips`?  Or do we not pre-size in golang?  Maybe `append` is smart.

Reply:
AFAIK no one does pre-sizing unless it's HFT and you fight for every milisecond. But then, you do not do golang ;-)

I will be happy to optimize this code (for the price that `append` will become uglier) if it ever proves to be inefficient

Reply:
Fair, I see no reason not to do it that way so let's have it